src/pages/Home/Home.jsx

```diff
@@ -1,8 +1,11 @@
-
+import LandingPage from '../LandingPage/LandingPage'
 const Home = () => {
   return (
-    <div>Home</div>
+    <div>
+      Home
+      <LandingPage></LandingPage>
```

Comment:
tutaj powinien być komponent Banner. LandingPage to osobna strona w której też będzie banner